docs/editor/intellisense.md

```diff
@@ -187,6 +187,16 @@ The key bindings shown below are the default key bindings. You can change these
 ]
 ```
 
+## Enhancing completions with AI
+
+In VS Code, you can enhance your coding with artificial intelligence (AI), such as suggestions for lines of code or entire functions, fast documentation creation, and help creating code-related artifacts like tests.
+
+[GitHub Copilot](https://copilot.github.com/) is an AI-powered code completion tool that helps you write code faster and smarter. You can use the [Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) in VS Code to generate code, or to learn from the code it generates.
+
+![Copilot extension in marketplace](./images/artifical-intelligence/copilot-extension.png)
+
+You can learn more about how to get started with Copilot in the [Copilot documentation](./artificial-intelligence.md).
```

Comment:
prefer to use the full relative path (/docs/editor/artificial-intelligence.md), also in links below and other sections

Reply:
Got it! Updated here and in all the docs.